src/git.zsh

```diff
@@ -245,6 +353,52 @@ _fzf_complete_git-unstaged-files_post() {
     done
 }
 
+_fzf_complete_git-remotes() {
+    local fzf_options="$1"
+    shift
+
+    _fzf_complete "--ansi --tiebreak=index $fzf_options" $@ < <(git remote --verbose | awk '
+        /\(fetch\)$/ {
+            gsub("\t", " ")
+            print
+        }
+    ' | _fzf_complete_git_tabularize)
+}
+
+_fzf_complete_git-remotes_post() {
+    awk '{ print $1 }'
+}
+
+_fzf_complete_git-refs() {
+    local fzf_options=$1
+    shift
+
+    local ref=${${$(git config remote.$repository.fetch 2> /dev/null)#*:}%\*}
+
+    _fzf_complete "--ansi --tiebreak=index $fzf_options" $@ < <(
+        git for-each-ref $ref --format='%(refname:short) %(contents:subject)' 2> /dev/null |
+        awk -v prefix=$prefix_option '{ print prefix $0 }' | _fzf_complete_git_tabularize
+    )
+}
+
+_fzf_complete_git-refs_post() {
+    input=$(cat)
+
+    if [[ -z $input ]]; then
+        return
+    fi
+
+    echo ${${input#*/}%% *}
+}
+
+_fzf_complete_git_specific_values() {
+    local fzf_options=$1
+    local values=$2
+    shift 2
+
+    _fzf_complete "--ansi --tiebreak=index $fzf_options" $@ < <(awk -v prefix=$prefix_option '{ print prefix $0 }' <<< ${values})
+}
+
```

Comment:
The name `specific_values` does not indicate whether it is user-specified or the constants. How about `_fzf_complete_git_constants`?

And let's not avoid touching `LBUFFER`. Add something like this to handle the case where the value ends with `=`:
```zsh
_fzf_complete_git_constants_post() {
    local input=$(cat)

    if [[ $input = *= ]]; then
        echo -n $(cat)
    else
        echo <<< $input
    fi
}
```